client/checkout/woopay/express-button/woopay-express-checkout-button.js

```diff
@@ -316,6 +316,21 @@ export const WoopayExpressCheckoutButton = ( {
 		initWoopayRef.current = defaultOnClick;
 	}, [ defaultOnClick ] );
 
+	useEffect( () => {
+		const handlePageShow = ( event ) => {
+			// Re-enable the button after navigating back/forward to the page if bfcache is used.
+			if ( event?.persisted ) {
+				setIsLoading( false );
+			}
+		};
+
+		window.addEventListener( 'pageshow', handlePageShow );
+
+		return () => {
+			window.removeEventListener( 'pageshow', handlePageShow );
+		};
+	}, [] );
```

Comment:
Without this, and if back-forward cache is enabled in `chrome://flags`. The button is kept in its loading state after clicking the browser's back button.